### ISSUE TITLE
Passing true to the switch value is not working

### ISSUE THREAD

**Issue by bernsrd-systempsrtnets** [bcylin/QuickTableViewController#11]
As you can see I'm passing true value to the switch but it is not showing in the app it is on

```
Section(title: "App Settings", rows: [
                SwitchRow<SwitchCell>(title: "Push notifications", switchValue: true, icon: Icon(image: notification), action: weakify(self, type(of: self).didTogglePushNotification)),
                
                SwitchRow<SwitchCell>(title: "Enable touch ID", switchValue: isTouchIdenable, icon: Icon(image: touchid), action: weakify(self, type(of: self).didToggleTouchId))
                ])
```

**Comment by bcylin** [comment]
Can you provide the version you are using? This issue should be fixed in `0.8.2`.

**Comment by bernsrd-systempsrtnets** [comment]
sorry I made a mistake. In cellForRowAt I was making it off.
